config/zones/BE.yaml

```
@@ -45,9 +45,9 @@ capacity:
     - datetime: '2023-01-01'
       source: entsoe.eu
       value: 6475.0
-    - datetime: '2024-01-01'
-      source: entsoe.eu
-      value: 8789.0
+    - datetime: '2024-06-06'
+      source: elia.be
+      value: 9986.05
```

Keeping the old one:
```suggestion
    - datetime: '2024-01-01'
      source: entsoe.eu
      value: 8789.0
    - datetime: '2024-06-06'
      source: elia.be
      value: 9986.05
```